Frontend/src/cse-ui-kit/contentblock_popup/contentblock_popup.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { StyledContainer, StyledDottedContainer, StyledContent, MainText, BoldText } from './contentblock_popup-Styled';
+import { ReactComponent as ContentUpload } from 'src/cse-ui-kit/assets/upload-content.svg';
+
+type Props = {
+  children?: React.ReactElement | any;
+  onClick?: (...args: any) => void;
```

Comment:
Why is there an onclick function passed into the upload popup? Wouldnt onDrop make more sense?!? Or another event rather than onclick